conn.go

```diff
@@ -192,6 +195,11 @@ func (c *Conn) flate() bool {
 	return c.copts != nil
 }
 
+// RemoteAddr returns the remote address of websocket connection.
+func (c *Conn) RemoteAddr() string {
```

Comment:
I wonder if it would make sense to return a [`net.Addr`](https://pkg.go.dev/net#Addr) here, as it's a more well-known signature?

Reply:
if you think `net.Addr` is better, let me know so i can update it.

Reply:
The `RemoteAddr() string` signature is not in `net` or `x/net/websocket` AFAIK. If you look at the `x/net/websocket`, you can see that the return signature on Conn is `RemoteAddr() net.Addr`, not `string` as in this PR.

Reply:
ops, sorry. i thought it is its `.String()` method. yes, that's logical. let me update the pr.